templates/CRM/Core/BillingBlock.tpl

```diff
@@ -39,6 +39,14 @@
         {/foreach}
       </div>
     </fieldset>
+    {if $paymentAgreementTitle}
+      <div id="payment_notice">
+        <fieldset class="crm-public-form-item crm-group payment_notice-group">
+          <legend>{$paymentAgreementTitle}</legend>
+          {ts}Your account data will be used to charge your bank account via direct debit. While submitting this form you agree to the charging of your bank account via direct debit.{/ts}
```

Comment:
Shouldn't this be `{$paymentAgreementText}` and not hardcoded direct debit text?

Reply:
@mattwire yeah probably - just probably an oversight (but tbh I am probably not going to pick this up now cos I got the regression sorted & my todo list doesn't have space)

Reply:
Sorry we only tested the functional part and yes this should probably better be fixed that way – however I would also agree that the hard coded text should be ok for the moment ...

Reply:
It's a one line change, let's not leave inconsistency in the code: https://github.com/civicrm/civicrm-core/pull/26094

Reply:
It's on my todo but I am mostly afk for a few days.